contracts/BlastYield.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {IBlast, YieldMode as IBlast__YieldMode, GasMode as IBlast__GasMode} from "./interfaces/IBlast.sol";
+import {IBlastPoints} from "./interfaces/IBlastPoints.sol";
+import {IERC20Rebasing, YieldMode as IERC20Rebasing__YieldMode} from "./interfaces/IERC20Rebasing.sol";
+
+/**
+ * @title BlastYield
+ * @notice This contract is a base contract for future contracts that wish to claim Blast WETH or USDB yield to inherit from.
+ * @author LooksRare protocol team (👀,💎)
+ */
+contract BlastYield {
+    address public immutable WETH;
+    address public immutable USDB;
+
+    /**
+     * @param _blast Blast precompile
+     * @param _blastPoints Blast points
+     * @param _blastPointsOperator Blast points operator
+     * @param _owner Owner of the contract
+     * @param _usdb USDB address
+     * @param _weth WETH address
+     */
+    constructor(
+        address _blast,
+        address _blastPoints,
+        address _blastPointsOperator,
+        address _owner,
+        address _usdb,
+        address _weth
+    ) {
+        WETH = _weth;
+        USDB = _usdb;
+
+        IBlast(_blast).configure(IBlast__YieldMode.CLAIMABLE, IBlast__GasMode.CLAIMABLE, _owner);
+        IBlastPoints(_blastPoints).configurePointsOperator(_blastPointsOperator);
+        IERC20Rebasing(_weth).configure(IERC20Rebasing__YieldMode.CLAIMABLE);
+        IERC20Rebasing(_usdb).configure(IERC20Rebasing__YieldMode.CLAIMABLE);
+    }
+
+    /**
+     * @notice Claim Blast yield. Only callable by contract owner.
```

Comment:
this description is no longer true, you should say something like 'it is up to the inherited contract to decide how to guard this function'

Reply:
https://github.com/LooksRare/contracts-libs/pull/103/commits/e7dcbeb3b5c302ca7b2725b3cde6df60593d8056